CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Enhancements
 
 * Enhanced `Table.toString()` to show a PrimaryKey field details (#2903).
+* Ehhanced `RealmConfiguration.Builder` to have a constructor with Context and a custom folder path (#2900).
```

Comment:
Enable ReLinker when loading a Realm from a custom path by adding a `RealmConfiguration.Builder(Context, File)` constructor (#2900).

Also add a deprecation notice for the old constructor (we should remove that in 2.0)